packages/trader/src/Modules/Account/Containers/account.jsx

```diff
@@ -56,15 +49,6 @@ class Account extends React.Component {
         this.props.toggleAccount(true);
     }
 
-    componentDidUpdate(prevProps) {
```

Comment:
Can u confirm that this workaround is no longer needed? This was previously added to fix the refresh issue when account settings page is the route.

Reply:
@zaki-binary yes, added a fix for that by properly waiting for `account_status` before rendering the page. you can refer to this line here: https://github.com/binary-com/deriv-app/pull/419/files#diff-f61c6bb9ca5cc568f4f5c9cbb78d659cR551